beacon-chain/core/validators/validator_test.go

```diff
@@ -419,23 +419,23 @@ func TestSlashValidator_AlreadyWithdrawn(t *testing.T) {
 	}
 }
 
-func TestProcessPenaltiesExits_NothingHappened(t *testing.T) {
+func TestProcessSlashings_NothingHappened(t *testing.T) {
```

Comment:
ProcessSlashings has some tests but I don't believe I see any for ProcessExitQueue, can you add a test for its natural process and a test for an instance that it should fail please 😃?

Reply:
+1, Take a look at the coverage report, there are some major logic blocks that aren't tested. It may have already been this way, but would you mind adding a few quick tests to cover these conditions? Thanks!

https://codecov.io/gh/prysmaticlabs/prysm/compare/00ca1d7991197b40566d1c4a60d6985f97d42694...2b81d9a4896a9260be6529a3fbc232bc15da8db0/diff